.vscode/launch.json

```diff
@@ -61,7 +61,7 @@
         "--inspect-brk",
         "${workspaceRoot}/node_modules/.bin/jest",
         "--runInBand",
-        "--config=test/jest-ci.json",
+        "--config=test/jest-e2e.json",
```

Comment:
Not strictly related to the issue, but this `jest-ci.json` file doesn't existe anymore. I had to change this to debug the tests in local.